packages/kit/types/index.d.ts

```diff
@@ -506,7 +506,7 @@ export interface KitConfig {
 	typescript?: {
 		/**
 		 * A function that allows you to edit the generated `tsconfig.json`. You can mutate the config (recommended) or return a new one.
-		 * This is useful for — to example — extend a shared `tsconfig.json` in a monorepo root
+		 * This is useful to — fo example — extend a shared `tsconfig.json` in a monorepo root.
```

Comment:
obviously 'to example' was a typo but 'this is useful to' is not sensical english. i think we should change it to this

```suggestion
		 * This is useful for extending a shared `tsconfig.json` in a monorepo root, for example.
```